aquamarine/src/actor.rs

```diff
@@ -207,6 +215,7 @@ where
                 };
                 (reusables, res.effects, res.stats)
             }
+            .instrument(span)
```

Comment:
can we simplify it like that?
```suggestion
            .instrument(self.span.clone())
```

Reply:
done